### PR TITLE
Docs for new defaults

### PR DESCRIPTION
**Pull request by brandelune**
This PR keeps the doc related commits from https://github.com/omegat-org/omegat/pull/386

### CODE REVIEW

File: doc_src/OmegaT.css [hunk 1]
```diff
@@ -336,6 +334,18 @@ dd {
...
+
+#introduction\.make\.it\.look\.good ol>li::marker {
+	color: red;
+	font-weight:bold;
+}
+
+/***************************************/
+/* CSS for "docbook" coded screenshots */
+/***************************************/
+>>>>>>> 7da400f2d (number the UI sections on the screenshot + description):doc_src/OmegaT_new.css
```
Comment (miurahr) [inline doc_src/OmegaT.css:347]: It seems a merge failure of git operation

File: doc_src/en/Windows_Scripts.xml [hunk 2]
```diff
@@ -41,74 +41,48 @@
...
-	  <para>Scripts are also launched when you are executing other scripts. That
-	  means that on a large project, an "entry_activated" script will be called
-	  a lot of time when using a search/replace type of script that loops
-	  through all the segments, rendering the application unresponsive. </para>
-	</warning>
+  <para>Some scripts are <emphasis>event</emphasis>-based and can be set to
+  trigger automatically when a given event takes place. Each event corresponds
+  to a subfolder under <link
+  linkend="application.folder.scripts"
+  endterm="application.folder.scripts.title"/>:</para>
```
Comment (miurahr) [inline doc_src/en/Windows_Scripts.xml:48]: Only ` :` has a link

File: doc_src/en/Windows_Scripts.xml [hunk 3]
```diff
@@ -41,74 +41,48 @@
...
+
+  <variablelist>
+	<varlistentry id="application.folder.scripts.application.shutdown">
+	  <term id="application.folder.scripts.application.shutdown.title">application_shutdown</term>
+	  <listitem><para>Scripts in this folder are launched before OmegaT shuts down.</para></listitem>
```
Comment (miurahr) [inline doc_src/en/Windows_Scripts.xml:53]: It seems no contents change but  indent and spacing are changed. Is it intended changes?

**Review by miurahr — COMMENTED** [review]
There are several weird place in changes.

**Comment by brandelune** [comment]
Thank you for checking already. I noticed after rebasing that some items here had issues. I'll fix them locally and push.

**Comment by brandelune** [comment]
I just pushed on my branch but the fix does not appear here...

**Comment by brandelune** [comment]
We should be good now. It would be nice if @Kazephil could check the contents because the rebase/merging process may have left things on the side.

**Comment by Kazephil** [comment]
I'll taken a look later this evening.

**Review by Kazephil — COMMENTED** [review]
@brandelune 
The rebase pretty good. Take a look I the changes I've proposed. If you're happy with them, I'll merge them as soon as possible.

**Review by brandelune — COMMENTED** [review]
Are you sure that the modifications are intended?

**Comment by Kazephil** [comment]
> Are you sure that the modifications are intended?

They are now :grinning: !

**Comment by brandelune** [comment]
> > Are you sure that the modifications are intended?
> 
> They are now 😀 !

Maybe in your future they are and I'm still in your past... ;-)

**Comment by Kazephil** [comment]
> Maybe in your future they are and I'm still in your past... ;-)

Ah, the "T" in OmegaT stands for "time travel". Clever!
My present self noticed a few things in the that my past self missed. And your near past self added material in the introduction that wasn't there when my further past self last read it. :)

**Comment by Kazephil** [comment]
@brandelune 
Are you happy with this one? I'd like to merge it before moving to the UI PR.

**Comment by brandelune** [comment]
I think there are 2 review items above that you have not yet solved.

**Comment by Kazephil** [comment]
> I think there are 2 review items above that you have not yet solved.

Oops. Take a look again.

**Comment by Kazephil** [comment]
@miurahr, @brandelune, @t-cordonnier 

I just tried to merge this, but when I run `./gradlew check`, I get a lot of `unchecked conversion` warnings and the check stops with  `error: warnings found and -Werror specified`.

This happens not just on this branch, but also on my local copy of the latest SourceForge master as well as on the latest copies on my on fork. Does anyone else have the same issue?

**Comment by brandelune** [comment]
> @miurahr, @brandelune, @t-cordonnier
> 
> I just tried to merge this, but when I run `./gradlew check`, I get a lot of `unchecked conversion` warnings and the check stops with `error: warnings found and -Werror specified`.
> 
> This happens not just on this branch, but also on my local copy of the latest SourceForge master as well as on the latest copies on my on fork. Does anyone else have the same issue?

Indeed, lots of warnings, and errors in two files.



Name | Errors
-- | --
omegat.org/test-integration/src/org/omegat/core/data/TestTeamIntegrationChild.java | 21
omegat.org/test-integration/src/org/omegat/core/data/TestTeamIntegration.java | 18

**Comment by Kazephil** [comment]
> Indeed, lots of warnings, and errors in two files.

This seems to have happened after the merge of the PR that removes the TaaS references.
The check passes in the branch I haven't updated to that merge yet.

**Comment by Kazephil** [comment]
Merged into `master`.

**Pull request closed without merging**